Formula/blast.rb

```diff
@@ -40,13 +40,17 @@ class Blast < Formula
   def install
     cd "c++" do
       # Boost is only used for unit tests.
-      args = %W[--prefix=#{prefix}
-                --with-bin-release
-                --with-mt
-                --with-strip
-                --with-experimental=Int8GI
-                --without-debug
-                --without-boost]
+      args = %W[
+        --prefix=#{prefix}
+        --with-bin-release
+        --with-mt
+        --with-strip
+        --with-experimental=Int8GI
+        --without-debug
+        --without-boost
+      ]
```

Comment:
```suggestion
      args = *std_configure_args + %W[
        --with-bin-release
        --with-mt
        --with-strip
        --with-experimental=Int8GI
        --without-debug
        --without-boost
      ]
```

Reply:
I recall trying this and the `configure` script doesn't like unknown arguments.

From `configure.ac`:
```
      * )
      AC_MSG_ERROR([$x_arg:  unknown option;  use --help to show usage])
```